src/input-select/input-select.styles.tsx

```diff
@@ -190,3 +190,10 @@ export const ValueLabel = styled(Text.Body)<ValueLabelStyleProps>`
 export const PlaceholderLabel = styled(ValueLabel)`
     color: ${Color.Neutral[3]};
 `;
+
+export const RangeIcon = styled.div`
```

Comment:
Did the flex in `Selector` not work?

Reply:
Actually I tried using `ArrowRangeIcon` from date input, but can't be used as can not nest a button within a button. And yes, without this styling it did not render correctly within selector.